src/gsy_e/gsy_e_core/simulation/setup.py

```diff
@@ -0,0 +1,95 @@
+"""
```

Comment:
Nit: I'm not sure, but maybe calling this module `simulation_setup.py` would be better to respect the naming of the class (making it easier to find). I know this leads to the repetition of the namespace (i.e. `simulation.simulation_setup`) and that's why I'm not completely sure this is a good practice. But I couldn't find anything about it. What's your opinion?

Reply:
I just saw the comment from spyros about the modules' names ([here](https://github.com/gridsingularity/gsy-e/pull/1533#pullrequestreview-1109126282)). If you've already agreed on that, that's good to go.